ran-simulator/templates/service.yaml

```diff
@@ -21,4 +21,8 @@ spec:
       port: 5150
     - name: exporter
       port: 9090
+    {{- range $i, $j := until (int .Values.towerPort.numTowers) }}
+    - name: tower-{{ $i }}
```

Comment:
the `ran-simulator` application itself creates these ports on the service from 
https://github.com/onosproject/ran-simulator/blob/c17434a548562cb524bdffa7eeead4e739a7c5eb/pkg/southbound/kubernetes/kubernetes.go#L38
so they should not be added here

Reply:
Hmm.. without this, ONOS-RIC did not receive any messages from RAN-Simulator. Let me double check.

Reply:
Hmm - did you check what the service looked like after ran-simulator added the ports? Is there a problem with the container name or something?

Reply:
I think so. Let me check it.

Reply:
I removed this part. I resolved that issue.